### ISSUE TITLE
Flops count

### ISSUE THREAD

**Issue by voldemortX** [facebookresearch/fvcore#69]
I notice in the [doc](https://detectron2.readthedocs.io/en/latest/modules/fvcore.html#fvcore.nn.FlopCountAnalysis) that "We count one fused multiply-add as one flop.", so does "flop" here means the commonly used "MAC"?

**Comment by ppwwyyxx** [comment]
Yeah it actually means MAC. There is a long-standing confusion as many papers use "flop" to mean "MAC". 

Ideally we'd like to fix the inconsistency but that will break some users of us.

**Comment by voldemortX** [comment]
Thanks for the clarifications. I think for now most users can guess it is MAC after reading the documentation.
